Sources/Sentry/SentryDevice.mm

```diff
@@ -18,8 +18,6 @@
 #import <sys/sysctl.h>
 #if SENTRY_HAS_UIKIT
 #    import <UIKit/UIKit.h>
-#elif TARGET_OS_WATCH
-#    import <WatchKit/WatchKit.h>
```

Comment:
It kinda dont make sense. 
Apple own header have errors inside, maybe this will be solved until Xcode 15 release, but we don't even use this, so there is no harm in removing it and save a few bits.